### PR TITLE
add django app for GCE screener responses

### PR DESCRIPTION
**Pull request by austensen**
Adds a new app to save user responses on the standalone Good Cause Eviction screener tool. We set up the model for responses, and a single API endpoint for POST requests from the screener. On first the first request with address it creates a new record and returns the ID that's autogenerated in the DB. On subsequent requests with that ID included it will update the fields with new data. 

Since all data being sent it directly controlled by us on the GCE screener side (no direct user input strings), I don't think we need extensive validation with forms.py, and it wasn't working as it does in wow), so we use a simple pydantic model with basic tyes. But in future if we wanted we can use pydantics validation options to directly match the django model (eg. options for yes/no/unsure, etc.) 

Since we are not using the frontend of tenants2, we need to log errors for 500 responses so they get tracked by rollbar. 

I have the GCE_ORIGN set in justfix_environment.py with the temporary one we are using, but we'll need to update this once we settle on a final domain for the screener. 

Also, unrelated to this project, I added port forwarding for the postgres DB in docker-compose so we can connect locally (eg. via postico) for easier debugging.

Companion PR for gce-screener: https://github.com/JustFixNYC/gce-screener/pull/39

[sc-15889]
[sc-15887]
[sc-15913]
[sc-15915]

### CODE REVIEW

File: gce/util.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    elif len(auth) > 2:
+        raise AuthorizationError("Invalid token header. Token string should not contain spaces.")
+
+    request_token = auth[1]
+
+    if not (token == request_token):
+        raise AuthorizationError("You do not have permission to access this resource")
+
+
+def apply_cors_policy(response):
```
Comment (kiwansim) [inline gce/util.py:41]: this is neat that you were able to just apply the headers here

**Review by kiwansim — APPROVED** [review]

File: gce/util.py (hunk 1 above)
Comment (austensen) [reply gce/util.py:41]: yeah I was pleasantly surprised this was possible too. We just needed to also cover the `OPTIONS` requests with it too for the preflight requests

File: gce/models.py [hunk 2]
```diff
@@ -108,6 +76,15 @@ class GoodCauseEvictionScreenerResponse(models.Model):
         null=True,
     )
 
+    form_answers_final = models.JSONField(
```
Comment (kiwansim) [inline gce/models.py:79]: how would we handle the case of a user retaking the survey more than once? would we continue to override the `form_answers_final` field?

**Review by kiwansim — APPROVED** [review]

File: gce/models.py (hunk 2 above)
Comment (austensen) [reply gce/models.py:79]: yeah, that's what I was thinking. but good point that maybe there's something useful in knowing each round of taking it. I was just thinking that we'd only want to know the final set of answers after they might take a couple passes at if after following the guide pages. I can check with Joel to see if that's enough for what we need

Comment (austensen) [reply gce/models.py:79]: confirmed this is ok with joel

**Pull request merged**